www/http/server.go

```diff
@@ -34,18 +38,26 @@ func NewServer(conf *Config) *Server {
 	}
 }
 
-func (s *Server) StartServer(capnpServer string) error {
+func (s *Server) StartServer(grpcServer string) error {
 	if !s.config.Enable {
 		return nil
 	}
 
-	c, err := net.Dial("tcp", capnpServer)
+	c, err := grpc.DialContext(
```

Comment:
```suggestion
	conn, err := grpc.DialContext(
```

I suggest give a better name to variable `c`

Reply:
Integrated into next force push.